waltz-ng/client/flow-diagram/components/editor/flow-diagram-editor.js

```diff
@@ -146,6 +148,7 @@ function mkDisjointNodeMenu($timeout, vm, flowDiagramStateService) {
                         };
                         vm.popup = popup;
                         vm.visibility.disjointNodePopup = true;
+                        vm.visibility.anyPopup = true;
```

Comment:
perhaps `vm.visibility.anyPopup = () => vm.visibility.disjointNodePopup || vm.visibility.diagramInfoPopup;` to avoid potential forgetfulness errors

Reply:
there are about 6 popups now!

Reply:
than the lamba is probably a good way to go, it can be put into initial state and you know it will update based on the constituent popups